HW03/veaceslav_irizanov/veaceslav_irizanov_hw_03_t_03.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require 'time'
+
+class Homework3
+  def task3(log)
+    rasult_array = time_line(total_rows(log))
+    final_array = array_of_duration(rasult_array)
+    final_array.size == 1 ? final_array[0] : final_array
```

Comment:
no need to create a variable for this 
```suggestion
    final_array = array_of_duration(time_line(total_rows(log)))
    final_array.size == 1 ? final_array[0] : final_array
```